### PR TITLE
restructure of the memory caching document

### PR DESCRIPTION
**Pull request by voroyam**
removed instructions of other distributions other than Ubuntu.

added a quick configuration example.

fixes https://github.com/owncloud/docs/issues/2349

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/caching_configuration.adoc [hunk 1]
```diff
@@ -107,16 +102,7 @@ because it guarantees that cached objects are available for as long as they are
 The Redis PHP module must be at least version 2.2.6 or higher.
 If you are running a Linux distribution that does not package the supported versions of this module — or does not package Redis at all — see xref:installing-redis-on-other-distributions[Installing Redis on other distributions].
 
-TIP: Debian Jessie users, please see this https://github.com/owncloud/core/issues/20675#issuecomment-159202901[GitHub discussion] if you have problems with LDAP authentication when using Redis.
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/server/caching_configuration.adoc]: Would it not make sense to keep this line?

File: modules/admin_manual/pages/configuration/server/caching_configuration.adoc [hunk 2]
```diff
@@ -47,6 +47,31 @@ The cache directory defaults to `data/$user/cache` where `$user` is the
...
 (See config_sample_php_parameters) to select a different location.
 
+== Quick Configuration Example
+
+Install APCu and Redis:
+
+[source,console]
+----
+apt install php-apcu
+apt install redis-server php-redis
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/server/caching_configuration.adoc]: These two lines could be combined into `apt install php-apcu redis-server php-redis`

File: modules/admin_manual/pages/configuration/server/caching_configuration.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/server/caching_configuration.adoc]: Not if we agreed to make the docs ubuntu only.

Additionally:

> Debian “jessie” Release Information
> Debian 8.11 was released June 23rd, 2018. Debian 8.0 was initially released on April 26th, 2015. The release included many major changes, described in our press release and the Release Notes.
> 
> Debian 8 has been superseded by Debian 9 (stretch). Regular security support updates have been discontinued as of June 17th, 2018.
> 
> Jessie also benefits from Long Term Support (LTS) until the end of June 2020.

File: modules/admin_manual/pages/configuration/server/caching_configuration.adoc (hunk 2 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/server/caching_configuration.adoc]: or we keep them separated. If one fails you know what failed. If the combined line fails you are not sure if it failed bacause of apcu or redis sever or php-redis.

File: modules/admin_manual/pages/configuration/server/caching_configuration.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/server/caching_configuration.adoc]: I thing that it should be differenciated between describing commands based on Ubuntu vs giving helpful hints for other distros where not the command is in forground but the tip.

File: modules/admin_manual/pages/configuration/server/caching_configuration.adoc (hunk 2 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/caching_configuration.adoc]: Afaik the output of `apt install` shows which packages are not able to be installed and why, such as a dependent package is not installable or held back.

File: modules/admin_manual/pages/configuration/server/caching_configuration.adoc (hunk 1 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/caching_configuration.adoc]: > Not if we agreed to make the docs ubuntu only.
> 
> Additionally:
> 
> > Debian “jessie” Release Information
> > Debian 8.11 was released June 23rd, 2018. Debian 8.0 was initially released on April 26th, 2015. The release included many major changes, described in our press release and the Release Notes.
> > Debian 8 has been superseded by Debian 9 (stretch). Regular security support updates have been discontinued as of June 17th, 2018.
> > Jessie also benefits from Long Term Support (LTS) until the end of June 2020.

Fair enough.

File: modules/admin_manual/pages/configuration/server/caching_configuration.adoc (hunk 2 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/server/caching_configuration.adoc]: -.- okay, will change. faster than having this discussion.

**Pull request merged**